src/Stratis.Bitcoin.IntegrationTests/API/ApiSpecification.cs

```diff
@@ -177,5 +177,29 @@ public void Status_returns_status_info()
             When(calling_status);
             Then(status_information_is_returned);
         }
+
+        [Fact]
+        public void POS_node_calls_getstakinginfo_returns_info()
+        {
+            Given(a_proof_of_stake_node_with_api_enabled);
+            When(calling_getstakinginfo);
+            Then(staking_information_is_returned);
+        }
+
+        [Fact]
+        public void POW_node_calls_getstakinginfo_and_receives_error()
+        {
+            Given(a_pow_node_with_api_enabled);
```

Comment:
to be consistent with the other Givens,  this should be `a_proof_of_work_node....`  or the  `a_proof_of_stake_node...`  should be `a_pos_node...`    I favour the long form as it is easier for anyone new to the code.

Reply:
Sounds good, will update accordingly.